test.tex

```diff
@@ -4,7 +4,10 @@
 
 \title{Ceci est un titre}
 \author{Karnaj, Clem}
-\licence{CC-BY-NC-ND}
+\licence[./img/license/cc-by-nc-nd_icon.svg]{CC-BY-NC-ND}{https://creativecommons.org/licenses/by-nc-nd/4.0/legalcode}
+
+% \logo{logo-tuto.png}
+% \tutoLink{https://link-to-tuto.com/}
```

Comment:
Maybe avoid commenting lines that may be important ^^

(remove them if you think they are not)

Reply:
And add this in documentation ?

Reply:
Yes :)